lib/src/st2110/st_fmt.c

```diff
@@ -57,6 +57,13 @@ static const struct st20_pgroup st20_pgroups[] = {
         .coverage = 4,
         .name = "ST20_FMT_YUV_420_12BIT",
     },
+    {
+        /* ST20_FMT_YUV_420_16BIT */
+        .fmt = ST20_FMT_YUV_420_16BIT,
+        .size = 12,
```

Comment:
why magic number not `#define` or enum like a line above ?

Reply:
no idea, I did not created this table, just following the existing flow, but I think THIS table is the define you are asking